doc/api/util.md

```diff
@@ -417,6 +417,13 @@ changes:
     objects the same as arrays. Note that no text will be reduced below 16
     characters, no matter the `breakLength` size. For more information, see the
     example below. **Default:** `true`.
+  * `minComplexity` {integer} The minimum complexity up to which an object is
+    definitely going to be fully inspected. This corresponds roughly to the
+    number of properties that are inspected. If the object exceeds that
+    complexity while inspecting, the inspection is continued up to one more
+    second. If the inspection does not complete in that second, it will limit
+    all further inspection to the absolute minimum and an `INSPECTION_ABORTED`
+    warning is emitted. **Default:** `1e5`.
```

Comment:
I am fine with setting this to Infinity first and then to change the default.

However, I would really prefer to have a single option for this. We already have quite a few options and IMO setting this super fine grained does not really help. I guess we could already call this `maxComplexity` instead of `min` while keeping the functionality as is and just rewording it a bit.

We could also bind the extra time to the requested complexity as in timeout after `Math.floor(opts.complexity / 100)` ms.

Reply:
I think the rename to `maxComplexity` is a good idea.

I do think we should always provide *some* way to limit the complexity while still getting absolutely deterministic results (i.e. no timer). This PR optimizes for the common case of Node.js usage, but for some applications determinism is a paramount property and we should not disable those use cases.

I agree that we already a larger number of options here than what would be ideal, but I think making these two options would be the best approach atm.